### ISSUE TITLE
[Bug Report] Could not initialize lavaplayer.

### ISSUE THREAD

**Issue by vastofficial** [jagrosh/MusicBot#451]
**Describe the bug**
Could not initialize lavaplayer.

**To Reproduce**
1play https://www.youtube.com/playlist?list=PL0hL1nfWyT5Hmo8gB9N00GuKaIoqT46O9

**Expected behavior**
Just normally load the playlist and play it like it did in the past..

**Screenshots**
[06:25:50] [ERROR] [DefaultAudioPlayerManager]: Error in loading item https://www.youtube.com/playlist?list=PL0hL1nfWyT5Hmo8gB9N00GuKaIoqT46O9
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:237)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:226)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.executeQueue(OrderedExecutor.java:98)
at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.run(OrderedExecutor.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.sedmelluq.discord.lavaplayer.tools.JsonBrowser
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubePlaylistLoader.load(DefaultYoutubePlaylistLoader.java:41)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.playlist(YoutubeAudioSourceManager.java:232)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.playlist(YoutubeAudioSourceManager.java:220)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.routeFromMainDomain(DefaultYoutubeLinkRouter.java:67)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:39)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:188)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:127)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218)

**Debug Info**
```
System Properties:
  java.version = 1.8.0_232
  java.vm.name = OpenJDK 64-Bit Server VM
  java.vm.specification.version = 1.8
  java.runtime.name = OpenJDK Runtime Environment
  java.runtime.version = 1.8.0_232-b09
  java.specification.version = 1.8
  os.arch = amd64
  os.name = Linux

JMusicBot Information:
  Version = 0.2.9
  Owner = 141302663022116864
  Prefix = 1
  AltPrefix = null
  MaxSeconds = 0
  NPImages = false
  SongInStatus = true
  StayInChannel = false
  UseEval = false
  UpdateAlerts = true

Dependency Information:
  JDA Version = 3.8.3_464
  JDA-Utilities Version = 2.1.5
  Lavaplayer Version = 1.3.49

Runtime Information:
  Total Memory = 26
  Used Memory = 21

Discord Information:
  ID = 658610915461169162
  Guilds = 1
  Users = 157
```

**Additional context**

**Comment by Sanduhr32** [comment]
Is the playlist public & accessible for anonymous access without being logged in, no geo & age restrictions?

**Comment by vastofficial** [comment]
Nope, I still get the error loading a normal video..
https://www.youtube.com/watch?v=uAYG46w1SCA

OpenJDK Runtime Environment (build 1.8.0_232-b09)
OpenJDK 64-Bit Server VM (build 25.232-b09, mixed mode)
:/home/container$ java -Djavax.accessibility.assistive_technologies= -Dnogui=true -jar JMusicBot.jar
[20:07:23] [INFO] [Startup]: Loaded config from /home/container/config.txt
[20:07:23] [INFO] [JDA]: Login Successful!
[20:07:24] [INFO] [WebSocketClient]: Connected to WebSocket
[20:07:24] [INFO] [JDA]: Finished Loading!
[20:07:57] [ERROR] [DefaultAudioPlayerManager]: Error in loading item https://www.youtube.com/watch?v=uAYG46w1SCA
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:237)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:226)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.executeQueue(OrderedExecutor.java:98)
at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.run(OrderedExecutor.java:87)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/exc/MismatchedInputException
at com.sedmelluq.discord.lavaplayer.tools.JsonBrowser.setupMapper(JsonBrowser.java:217)
at com.sedmelluq.discord.lavaplayer.tools.JsonBrowser.<clinit>(JsonBrowser.java:21)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.load(DefaultYoutubeTrackDetailsLoader.java:45)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetailsLoader.loadDetails(DefaultYoutubeTrackDetailsLoader.java:26)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadTrackWithVideoId(YoutubeAudioSourceManager.java:200)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.track(YoutubeAudioSourceManager.java:224)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.track(YoutubeAudioSourceManager.java:220)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.routeFromUrlWithVideoId(DefaultYoutubeLinkRouter.java:96)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.routeFromMainDomain(DefaultYoutubeLinkRouter.java:61)
at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:39)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:188)
at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:127)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419)
at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218)
... 6 common frames omitted
Caused by: java.lang.ClassNotFoundException: com.fasterxml.jackson.databind.exc.MismatchedInputException
at java.net.URLClassLoader$1.run(URLClassLoader.java:371)
at java.net.URLClassLoader$1.run(URLClassLoader.java:363)
at java.security.AccessController.doPrivileged(Native Method)
at java.net.URLClassLoader.findClass(URLClassLoader.java:362)
at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
... 21 common frames omitted
Caused by: java.util.zip.ZipException: invalid LOC header (bad signature)
at java.util.zip.ZipFile.read(Native Method)
at java.util.zip.ZipFile.access$1400(ZipFile.java:60)
at java.util.zip.ZipFile$ZipFileInputStream.read(ZipFile.java:734)
at java.util.zip.ZipFile$ZipFileInflaterInputStream.fill(ZipFile.java:434)
at java.util.zip.InflaterInputStream.read(InflaterInputStream.java:158)
at sun.misc.Resource.getBytes(Resource.java:124)
at java.net.URLClassLoader.defineClass(URLClassLoader.java:463)
at java.net.URLClassLoader.access$100(URLClassLoader.java:74)
at java.net.URLClassLoader$1.run(URLClassLoader.java:369)
... 27 common frames omitted

**Comment by jagrosh** [comment]
Which jar from the releases page are you using (which OS)?

**Comment by vastofficial** [comment]
John, isn't that why I provided the debug info?!? xD

**Comment by jagrosh** [comment]
Because of how I package the jars, the debug info can't actually show if you're using the multi-platform jar or the specific OS one

**Comment by vastofficial** [comment]
I think I'm using the linux one, but I'm running it in a docker instance on pterodactyl

**Comment by vastofficial** [comment]
Maybe I should use the the universal one??

**Comment by jagrosh** [comment]
I took a look at the releases for 0.2.9 and none of them should be having this issue. This seems like either docker or pterodactyl isn't working correctly because it's failing to read the contents of the jar.

**Comment by vastofficial** [comment]
Okay I got it!!! I didn't use the universal version, that one works!!!!

**Comment by vastofficial** [comment]
I found a different issue tho, search engine seems broken:
1play doja cat say so
returns
No results found for doja cat say so

**Comment by jagrosh** [comment]
That's already addressed by #443 and fixed in [version 0.2.10](https://github.com/jagrosh/MusicBot/releases/latest)
